frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -18,39 +20,54 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
   });
 
   const handleFormSubmit = useCallback(
-    (event_: React.BaseSyntheticEvent): void => {
-      void handleSubmit(onSubmit)(event_);
+    (event: React.BaseSyntheticEvent): void => {
+      void handleSubmit(onSubmit)(event);
```

Comment:
return this back please, we have global object event
```suggestion
      void handleSubmit(onSubmit)(event_);
```

Reply:
done